### PR TITLE
Amend the list of ipvs schedulers with descriptions

### PR DESCRIPTION
**Pull request by tengqm**
This PR fixes the IPVS scheduler name list by adding schedulers that were missing and adding some descriptions (copied for source code comments) for each.

### CODE REVIEW

File: content/en/docs/reference/networking/virtual-ips.md [hunk 1]
```diff
@@ -222,12 +221,41 @@ higher throughput of network traffic.
...
+  If the target set has not changed for the specified time, the most loaded server
+  is removed from the set, in order to avoid high degree of replication.
+
+* `sh` (Source Hashing): Traffic is sent to a backing server by looking up a statically
+  assigned hash table based on the source IP addresses.
+
+* `dh` (Destination Hashing): Traffic is sent to a backing server by looking up a
+  statically assigned hash table based on their destination addresses.
+
+* `sed` (Shhortest Expected Delay): traffic forwarded to a backing server with the shortest
```
Comment (kbhawkey) [inline content/en/docs/reference/networking/virtual-ips.md:251]: nit: capitalize Traffic

**Comment by Shubham82** [comment]
@tengqm Thanks for this PR.
The test cases failed due to Netlify build timeout failures.
please re-push the commit.

File: content/en/docs/reference/networking/virtual-ips.md [hunk 2]
```diff
@@ -222,12 +221,41 @@ higher throughput of network traffic.
...
+  If the target set has not changed for the specified time, the most loaded server
+  is removed from the set, in order to avoid high degree of replication.
+
+* `sh` (Source Hashing): Traffic is sent to a backing server by looking up a statically
+  assigned hash table based on the source IP addresses.
+
+* `dh` (Destination Hashing): Traffic is sent to a backing server by looking up a
+  statically assigned hash table based on their destination addresses.
+
+* `sed` (Shhortest Expected Delay): Traffic forwarded to a backing server with the shortest
```
Comment (reylejano) [inline content/en/docs/reference/networking/virtual-ips.md:251]: typo
```suggestion
* `sed` (Shortest Expected Delay): Traffic forwarded to a backing server with the shortest
```

Comment (tengqm) [reply content/en/docs/reference/networking/virtual-ips.md:251]: Fixed.

**Pull request merged**